interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -147,57 +147,104 @@ components:
       required:
         - uomDimensions
         - verticalReferenceType
-        - verticalReference
-        - longitude
-        - latitude
```

Comment:
Why did you remove the requirement on those fields?

Reply:
[To accommodate serializers that can't differentiate between omission and the default value](https://github.com/interuss/dss/pull/818#discussion_r993964212)